.github/workflows/ci.yml

```diff
@@ -48,7 +48,7 @@ jobs:
           path: ~/.gradle/caches
           key: gradle-${{ runner.os }}-${{ hashFiles('**/build.gradle') }}-${{ hashFiles('**/gradle/wrapper/gradle-wrapper.properties') }}
       - name: Unit tests
-        run: ./gradlew testDebugUnitTest
+        run: ./gradlew :stripe:testDebugUnitTest
```

Comment:
👍

Reply:
We have to be careful to remember to add in any new modules

The other thing I considered was specifically excluding the tests we didn't want to run with -x :stripe-test-e2e:testDebugUnitTest.  This means that compiling of example and stripe-test-e2e will still happen, but might prevent forgetting to add new unit test modules?   🤷

Reply:
I can look into this.